### PR TITLE
German translation improvements from Crowdin made by:

### PR DESCRIPTION
**Pull request by magul**
* Raphael Das Gupta (https://crowdin.com/profile/das-g)

### CODE REVIEW

File: de/whats_next/README.md [hunk 1]
```diff
@@ -10,33 +10,33 @@ Und dann folge Django Girls doch auf [Facebook](http://facebook.com/djangogirls)
...
 
-- [Der Data Science Kurs von Codecademy](https://www.codecademy.com/learn/paths/data-science)
-- [edX](https://www.edx.org/course/?search_query=python&subject=Data%20Analysis%20%26%20Statistics) – die meisten Kurse sind kostenfrei, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
+- [Der Data-Science-Kurs von Codecademy](https://www.codecademy.com/learn/paths/data-science)
+- [edX](https://www.edx.org/course/?search_query=python&subject=Data%20Analysis%20%26%20Statistics) – die meisten Kurse kanst du kostenfrei testen, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
 - [Dataquest](https://www.dataquest.io/) – die ersten 30 "Missionen" sind frei
 
-Wir können es nicht abwarten zu sehen, was du als Nächstes baust!
\ No newline at end of file
+Wir gespannt darauf, was du als Nächstes baust!
```
Comment (das-g) [inline de/whats_next/README.md]: oops, missed the verb here. Will fix on Crowdin

Comment (das-g) [reply de/whats_next/README.md]: > Will fix on Crowdin

done

Comment (magul) [reply de/whats_next/README.md]: ...and Done!

File: de/whats_next/README.md [hunk 2]
```diff
@@ -10,33 +10,33 @@ Und dann folge Django Girls doch auf [Facebook](http://facebook.com/djangogirls)
...
+- [edX](https://www.edx.org/course?search_query=python) – die meisten Kurse kannst du kostenfrei testen, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
 - [Courseras Python-Kurse](https://www.coursera.org/specializations/python) – einige Video-Vorträge sind kostenlos
 
 #### Mit Daten arbeiten
 
-- [Der Data Science Kurs von Codecademy](https://www.codecademy.com/learn/paths/data-science)
-- [edX](https://www.edx.org/course/?search_query=python&subject=Data%20Analysis%20%26%20Statistics) – die meisten Kurse sind kostenfrei, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
+- [Der Data-Science-Kurs von Codecademy](https://www.codecademy.com/learn/paths/data-science)
+- [edX](https://www.edx.org/course/?search_query=python&subject=Data%20Analysis%20%26%20Statistics) – die meisten Kurse kanst du kostenfrei testen, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
```
Comment (normade) [inline de/whats_next/README.md]: "kanst" --> "kannst"

**Pull request merged**